packages/bruno-cli/src/commands/run.js

```diff
@@ -369,6 +372,28 @@ const handler = async function (argv) {
       );
 
       results.push(result);
+
+      // determine next request
+      const nextRequestName = result?.nextRequestName;
+      if (nextRequestName !== undefined) {
+        nJumps++;
+        if (nJumps > 10000) {
```

Comment:
Shouldn't it be a parameter in bruno.json ? just in case

Reply:
I thought about it -- but there are already a ton of config options... Do you have a use-case that needs it? If so, we can add it. If not, I would say YAGNI.

Reply:
YAGNI it is then ;)